ofl/notoserifmakasar/METADATA.pb

```diff
@@ -0,0 +1,22 @@
+name: "Noto Serif Makasar"
+designer: "Google"
+license: "OFL"
+category: "SERIF"
+date_added: "2022-10-11"
+fonts {
+  name: "Noto Serif Makasar"
+  style: "normal"
+  weight: 400
+  filename: "NotoSerifMakasar-Regular.ttf"
+  post_script_name: "NotoSerifMakasar-Regular"
+  full_name: "Noto Serif Makasar Regular"
+  copyright: "Copyright 2022 The Noto Project Authors (https://github.com/notofonts/makasar)"
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+source {
+  repository_url: "https://github.com/notofonts/makasar.git"
+  archive_url: "https://github.com/notofonts/makasar/releases/download/NotoSerifMakasar-v1.001/NotoSerifMakasar-v1.001.zip"
+}
+is_noto: true
```

Comment:
@simoncozens missing subset, languages and primary script.

Reply:
This is now in place, dependent on https://github.com/googlefonts/lang/pull/74 being merged.